404.html

```diff
@@ -0,0 +1,32 @@
+---
+title: 404 Error
+layout: default
+permalink: /404.html
+
+---
+<div class="use-grid-wide" >
+    <div class="usa-width-one-half usa-offset-one-fourth usa-section">
+
+    <h1>Error &nbsp 4 :o 4</h1> 
+    <div id="page-error"></div>
+    </div>
+</div>
+
+
+<script>
+
+function redirectToBlog() {
+    var url = window.location.toString();
+    if (url.includes('blog')) {
+        url = `https://codefordc.github.io/${url.slice(url.indexOf('blog'))}`;
+        document.getElementById('page-error')
+            .innerHTML = `<p>Hey, looks like this content has moved. Here is where you can find it: <a href='${url}'>${url}</a></p>`;
+    } else {
+        document.getElementById('page-error')
+            .innerHTML = `<p>Hey, looks like this page doesn't exist. <a href='https://codefordc.org/'>Back to the site!</a></p>`;
```

Comment:
```suggestion
            .innerHTML = `<p>Hey, looks like this page doesn't exist. <a href="{{ site.baseurl }}">Back to the site!</a></p>`;
```

This will then make the re-direct work regardless of where its getting run.

Reply:
I've been trying this out locally, but I can't seem to get it to work. The `baseurl` in the `_config.yml` file is and empty string, so the re-direct ends up directing users back to the 404 page. 

This 404 page is embedded into the actual website rather than being a separate page, so we could actually remove the `a` tag entirely since the user would still be on the page and can navigate via the navbar – or we can make the 404 page an entirely different page

Reply:
`url` in the config file is the actual url though, so maybe `site.url` instead of `site.baseurl`
```suggestion
            .innerHTML = `<p>Hey, looks like this page doesn't exist. <a href="{{ site.url }}">Back to the site!</a></p>`;
```